### PR TITLE
Added cancel auth feature

### PR DESCRIPTION
**Pull request by aenglander**

### CODE REVIEW

File: src/iovation.LaunchKey.Sdk/Error/AuthorizationResponseExists.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+﻿using System;
+namespace iovation.LaunchKey.Sdk.Error
+{
+    /// <summary>
+    /// Thrown when an action is attempted against an authorization request
+    /// byt cannot complete as the authorization request already has a respnse.
```
Comment (bporter125) [inline src/iovation.LaunchKey.Sdk/Error/AuthorizationResponseExists.cs]: response*

File: src/iovation.LaunchKey.Sdk/Transport/ITransport.cs [hunk 2]
```diff
@@ -27,6 +27,13 @@ public interface ITransport
         /// <returns>response details</returns>
         ServiceV3AuthsPostResponse ServiceV3AuthsPost(ServiceV3AuthsPostRequest request, EntityIdentifier subject);
 
+        /// <summary>
+        /// Cancels an existing and pending authorization request.
+        /// </summary>
+        /// <param name="authRequestId">the unique authorization request id for the request being canceled</param>
+        /// <param name="subject">the service ID associated with the auth reques</param>
```
Comment (bporter125) [inline src/iovation.LaunchKey.Sdk/Transport/ITransport.cs]: auth request*

File: src/iovation.LaunchKey.Sdk/Error/AuthorizationResponseExists.cs (hunk 1 above)
Comment (aenglander) [reply src/iovation.LaunchKey.Sdk/Error/AuthorizationResponseExists.cs]: Fixed

File: src/iovation.LaunchKey.Sdk/Transport/ITransport.cs (hunk 2 above)
Comment (aenglander) [reply src/iovation.LaunchKey.Sdk/Transport/ITransport.cs]: Fixed

File: src/iovation.LaunchKey.Sdk/Error/AuthorizationResponseExists.cs [hunk 3]
```diff
@@ -0,0 +1,26 @@
+﻿using System;
+namespace iovation.LaunchKey.Sdk.Error
+{
+    /// <summary>
+    /// Thrown when an action is attempted against an authorization request
+    /// bute cannot complete as the authorization request already has a response.
```
Comment (bporter125) [inline src/iovation.LaunchKey.Sdk/Error/AuthorizationResponseExists.cs]: but*

Comment (aenglander) [reply src/iovation.LaunchKey.Sdk/Error/AuthorizationResponseExists.cs]: Fixed

**Pull request merged**